ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
you don't have to create it, you can import the svg as normal. the loader creates a jsx object for you. 

to my mind, there is a paradigm shift in componentising which i think is brought out here, in that the implementation of `Logo` (or any import) becomes incidental to any instance of it. 

in this case, it _is_ an svg, but that shouldn't matter. _that_ it's an svg is contingent on an earlier implementation decision, not the design decision to display a logo here. it's just another component.

for example, the loader could indeed generate a raster fallback (not a bad idea actually), but you would never need to think about that. or we could change it to another format. you'd change the import path and that's it (we could be smarter and remove the extension if it was useful).